_targets.R

```diff
@@ -71,27 +71,31 @@ dummy_date <- "2021-12-31"
 # Define dataset of interest for the national geospatial fabric (used to fetch PRMS catchment polygons):
 gf_data_select <- 'GeospatialFabricFeatures_02.zip'
 
+## NLCD 
+# path all NLCD LC data 2001-2019
+NLCD_LC_path <-'/Users/msleckman/OneDrive - DOI/40.2 Data-driven salinity/Data/DRB-Inland/NLCDs_Final2/NLCDs_Final/'
```

Comment:
Local path added here. How can we pull from shared source? Caldera?

Reply:
Good question. Eventually we will be able to pull this in as a SB ID when the data release is made public. Once we're running on the HPCs, Caldera storage will be a good choice. In the interest of time, I'm okay with manually switching the local directory location. Any other preferences @jsadler2, @lekoenig?

Reply:
If we will rely on a local directory location for the NLCD data as a stopgap until the data release is public, I wonder if we can take advantage of the .Rproj file in our repo and specify the path as:
```
NLCD_LC_path <- paste0(getwd(),'1_fetch/in/NLCDs_Final') #or 
NLCD_LC_path <- paste0(here::here(),'1_fetch/in/NLCDs_Final')
```
We could indicate in `_targets.R` that you need to create a subdirectory in `1_fetch/in` called `NLCDs_Final` and copy the files from sharepoint into that folder locally.

Reply:
created a directory for NLCD_2001_2019 in path `1_fetch/in/NLCDs_2001_2019`  in `_targets.R` and fetch from new path in the `p1_NLCD_LC_data`.